IM_calculation/IM/read_waveform.py

```diff
@@ -118,7 +120,7 @@ def read_waveforms(
     path,
     bbseis,
     station_names=None,
-    comp=Ellipsis,
+    comp=tuple(Components)[:2],
```

Comment:
slicing a tuple?

Reply:
Is this just a fancy way of getting (Components.c090, Components.c000)? If so maybe just hardcode them, makes it much easier to read I recon.